lib/metadata/in_memory/backend.js

```diff
@@ -233,6 +240,14 @@ const metastore = {
                 return cb(errors.InvalidArgument);
             }
 
+            if (continuationToken && typeof continuationToken !== 'string') {
+                return cb(errors.InvalidArgument);
```

Comment:
Could be nice to add a customized description about these argument checks. I guess this depends on AWS behavior though.

Reply:
Yeah, AWS just returns the standard 'InvalidArgument' error. And it keeps in line with the other checks here to return the standard